### PR TITLE
feat: implement casbin middleware for poem

### PR DESCRIPTION
**Pull request by greenhandatsjtu**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+[package]
+name = "poem-casbin"
```
Comment (hackerchai) [inline Cargo.toml]: maybe  `poem-casbin-auth` is better

File: Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+version = "0.1.0"
+edition = "2021"
+license = "Apache-2.0"
+description = "Casbin poem access control middleware"
+homepage = "https://github.com/casbin-rs/poem-cabin"
+readme = "README.md"
+
+[lib]
+name = "poem_casbin"
```
Comment (hackerchai) [inline Cargo.toml]: same here

**Review by hackerchai — APPROVED** [review]
LGTM

File: Cargo.toml (hunk 1 above)
Comment (greenhandatsjtu) [reply Cargo.toml]: I've changed the name, thank you.

**Review by PsiACE — APPROVED** [review]
LGTM, Looking forward to seeing some tests and examples

**Comment by greenhandatsjtu** [comment]
> LGTM, Looking forward to seeing some tests and examples

Sure, I'll add them later.

**Pull request merged**